app/controllers/application_controller.rb

```diff
@@ -14,7 +14,7 @@ def e164_number(phone_number)
     if TelephoneNumber.valid?(phone_number, :gb)
       TelephoneNumber.parse(phone_number, :gb).e164_number
     else
-      TelephoneNumber.parse(phone_number).e164_number
+      TelephoneNumber.parse(phone_number.gsub(/^00/, "+")).e164_number
```

Comment:
Humm this regex is fairly easy to grok, but I generally find it can get difficult if we need to enhance it in the future.

Do you think there's any value to putting this in a constant somewhere with a sensible name and using that in the gsub here instead?

Ideally we could share that with the file below too... but if that's in `/lib` and this is in `/app`... that might be a pain?

Reply:
Yeah, I thought about making this into a constant but wasn't sure where we could put it.  It doesn't really seem like the right thing to be including in config.

I'll have a think about this (maybe put all the phone number stuff into `lib`), since it would be nice to have all the phone things in one place.